pkgs/by-name/hu/huion-switcher/package.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  nix-update-script,
+  fetchFromGitHub,
+  rustPlatform,
+  udev,
+  pkg-config,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "huion-switcher";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "whot";
+    repo = pname;
+    tag = version;
```

using finalAttrs make it easier to override
```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
  pname = "huion-switcher";
  version = "0.5.0";

  src = fetchFromGitHub {
    owner = "whot";
    repo = pname;
    tag = finalAttrs.version;
```

---

That would not work, consider: https://github.com/NixOS/nixpkgs/issues/310373

---

#310373 doesn't apply to places where version should be used, such as the fetching of the source code

---

fixed